.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -6,8 +6,6 @@ body:
       value: |
         Thank you for reporting an issue :pray:.
 
-        **Please only file issues for v11. Will will no longer fix issues for v10.** 
```

Comment:
-> We will

Reply:
That whole line is being deleted anyways, but good catch :D